diff --git a/dist/frontmatter.js b/dist/frontmatter.js
index 163f4009a1fb727e0a88d248d5b815ddb11a736b..8c0b3b3b3f5f8515808dd08d3227a035175a4015 100644
--- a/dist/frontmatter.js
+++ b/dist/frontmatter.js
@@ -1,4 +1,4 @@
-import yaml from "js-yaml";
+import { load as yamlLoad } from "js-yaml";
 import * as toml from "smol-toml";
 function isFrontmatterValid(frontmatter) {
   try {
@@ -14,7 +14,9 @@ function extractFrontmatter(code) {
   return frontmatterRE.exec(code)?.[1];
 }
 function getFrontmatterParser(code) {
-  return frontmatterTypeRE.exec(code)?.[1] === "+++" ? ["+++", toml.parse] : ["---", yaml.load];
+  return frontmatterTypeRE.exec(code)?.[1] === "+++"
+    ? ["+++", toml.parse]
+    : ["---", yamlLoad];
 }
 function parseFrontmatter(code, options) {
   const rawFrontmatter = extractFrontmatter(code);
